### PR TITLE
CSCwe98725 - Rocky linux OS name and vendor values are wrong

### PR DESCRIPTION
**Pull request by parthibann**
Changes to make HCL validation successful, os name and os vendor values now match with hcl ucs tool.

### CODE REVIEW

**Review by Ghufz — APPROVED** [review]
looks good to me

File: os-discovery-tool/rocky-os-name.sh
Comment (sa-sumanth) [inline os-discovery-tool/rocky-os-name.sh:1]: Can you please share a sample output of this?

Comment (parthibann) [reply os-discovery-tool/rocky-os-name.sh:1]: string present in the file: Rocky Linux release 8.7 (Green Obsidian)
Output: Rocky Linux 8.7

**Comment by sa-sumanth** [comment]
Looks good to merge

**Review by sa-sumanth — APPROVED** [review]

**Pull request merged**